com/ArchiveService.py

```diff
@@ -281,210 +315,287 @@ def verify_rinex_multiday(cnn, rinexinfo, Config):
         rnxlist.append(rnx.rinex)
         # some other file, move it to the repository
         retry_folder = os.path.join(Config.repository_data_in_retry,
-                                    'multidays_found/' + rnx.date.yyyy() + '/' + rnx.date.ddd())
+                                    'multidays_found/' + rnx.date.yyyy() +
```

Comment:
this style of line continuation is great, and very much our preference! It keeps the parameters aligned on subsequent lines, which is easy to read =)